src/Cake.Svn.Tests/Unit/Info/SvnInfoResultTests.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using Cake.Svn.Info;
+using Xunit;
+
+namespace Cake.Svn.Tests.Unit.Info
+{
+    public sealed class SvnInfoResultTests
+    {
+        public sealed class TheConstructor
+        {  
+            [Fact]
+            public void Should_Throw_If_RepositoryRoot_Is_Null()
+            {
+                // Given
+                // When
+                // Then
+                Assert.Throws<ArgumentNullException>("repositoryRoot", () =>
+                    new SvnInfoResult(
```

Comment:
You could a private helper method which creates a `SvnInfoResult` object to reduce code duplication